api-service/webapp-demo/src/app/app.component.ts

```diff
@@ -160,9 +162,17 @@ export class AppComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    // Extract ClientJobKey from hash if specified
     const parts = document.location.hash.substr(1).split('/');
     this.customClientJobKey = parts[0];
 
+    // Determine if page should render in embedded mode
+    var query = document.location.search.substr(1)
+    var embedded_query = /embedded=(true|false)/g.exec(query);
+    if (embedded_query) {
+      this.notEmbedded = (embedded_query[1] == 'true') ? false : true;
```

Comment:
You can just say !(embedded_query[1] == 'true') instead of this ternary expression.